department-of-veterans-affairs/caseflow
# Ki/APPEALS-42996 - Remove 3 Character Limitation for Search Bar

Resolves [APPEALS-42996 - Remove 3 Character Limitation for Search Bar](https://jira.devops.va.gov/browse/APPEALS-42996)

# Description
This removes the limitation of needing 3 characters before front-end page search happens.

## Acceptance Criteria

## Testing Plan

1. Login as AMBRISVACO and navigate to Correspondence Cases Page
    1. Repeat for JOLLY_POSTMAN - Your Correspondence Page)
2. Utilize Search Bar
    1. Search in any visible Column
    2. Switch between Tabs and validate search results still filtering correctly
    3. Switch between Pages and validate search results still filtering correctly
    4. Make sure Filter and Sorting still works with Search function
    5. Verify that it works with any text input -- 1 character, 2 characters, 3 or more characters
        1. Note: You will still get results for _any_ text matches in _any_ viewable column (i.e. - "11" will give you results for Veteran Details that have any occurrences of 11, a Date that has 11 for a month or day, etc.).

# Frontend
## User Facing Changes

![Screenshot 2024-04-01 at 11 47 07 AM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/1abd5113-4bf0-405b-96c9-7df9785be495)
![Screenshot 2024-04-01 at 11 52 12 AM](https://github.com/department-of-veterans-affairs/caseflow/assets/108023343/0b7881bf-1e8a-46e2-bd1d-118afdc4b4d1)


# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
